ATen/native (4/6): Use per-operator headers

peterbell10 opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* #75576
* #75575
* __->__ #75574
* #75573

Differential Revision: [D40126697](https://our.internmc.facebook.com/intern/diff/D40126697)

peterbell10 committed:
ATen/native (4/6): Use per-operator headers

[ghstack-poisoned]

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "ATen/native (4/6): Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "ATen/native (4/6): Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "ATen/native (4/6): Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Rebase and remove unneeded _meta.h ops headers on "ATen/native (4/6): Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "ATen/native (4/6): Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "ATen/native (4/6): Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Rebase on "ATen/native (4/6): Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"




[ghstack-poisoned]

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"




[ghstack-poisoned]

malfet commented:
@malfet has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D40126697).

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"


Differential Revision: [D40126697](https://our.internmc.facebook.com/intern/diff/D40126697)

[ghstack-poisoned]

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"


Differential Revision: [D40126697](https://our.internmc.facebook.com/intern/diff/D40126697)

[ghstack-poisoned]

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"


Differential Revision: [D40126697](https://our.internmc.facebook.com/intern/diff/D40126697)

[ghstack-poisoned]

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"


Differential Revision: [D40126697](https://our.internmc.facebook.com/intern/diff/D40126697)

[ghstack-poisoned]

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"


Differential Revision: [D40126697](https://our.internmc.facebook.com/intern/diff/D40126697)

[ghstack-poisoned]

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"


Differential Revision: [D40126697](https://our.internmc.facebook.com/intern/diff/D40126697)

[ghstack-poisoned]

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"


Differential Revision: [D40126697](https://our.internmc.facebook.com/intern/diff/D40126697)

[ghstack-poisoned]

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"


Differential Revision: [D40126697](https://our.internmc.facebook.com/intern/diff/D40126697)

[ghstack-poisoned]

peterbell10 committed:
Rebase and fix merge conflicts on "ATen/native (4/6): Use per-operator headers"


Differential Revision: [D40126697](https://our.internmc.facebook.com/intern/diff/D40126697)

[ghstack-poisoned]

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"


Differential Revision: [D40126697](https://our.internmc.facebook.com/intern/diff/D40126697)

[ghstack-poisoned]

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"


Differential Revision: [D40126697](https://our.internmc.facebook.com/intern/diff/D40126697)

[ghstack-poisoned]

peterbell10 committed:
Rebase on "ATen/native (4/6): Use per-operator headers"


Differential Revision: [D40126697](https://our.internmc.facebook.com/intern/diff/D40126697)

[ghstack-poisoned]

malfet commented:
@malfet has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D40126697).

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"


Differential Revision: [D40126697](https://our.internmc.facebook.com/intern/diff/D40126697)

[ghstack-poisoned]

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"


Differential Revision: [D40126697](https://our.internmc.facebook.com/intern/diff/D40126697)

[ghstack-poisoned]

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"


Differential Revision: [D40126697](https://our.internmc.facebook.com/intern/diff/D40126697)

[ghstack-poisoned]

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"


Differential Revision: [D40126697](https://our.internmc.facebook.com/intern/diff/D40126697)

[ghstack-poisoned]

peterbell10 committed:
Rebase on "ATen/native (4/6): Use per-operator headers"


Differential Revision: [D40126697](https://our.internmc.facebook.com/intern/diff/D40126697)

[ghstack-poisoned]

peterbell10 committed:
Remove assert from NestedTensorMath.cpp as its creating too many merge conflicts on "ATen/native (4/6): Use per-operator headers"


Differential Revision: [D40126697](https://our.internmc.facebook.com/intern/diff/D40126697)

[ghstack-poisoned]

malfet commented:
@malfet has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D40126697).

peterbell10 committed:
Rebase on "ATen/native (4/6): Use per-operator headers"


Differential Revision: [D40126697](https://our.internmc.facebook.com/intern/diff/D40126697)

[ghstack-poisoned]

peterbell10 committed:
Update on "ATen/native (4/6): Use per-operator headers"


Differential Revision: [D40126697](https://our.internmc.facebook.com/intern/diff/D40126697)

[ghstack-poisoned]

malfet commented:
@malfet has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D40126697).
